de.z_ray.OptimusUI.json

```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "de.z_ray.OptimusUI",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "optimus-ui",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--talk-name=org.freedesktop.Flatpak"
```

Comment:
please submit an exception for this https://docs.flathub.org/docs/for-app-authors/linter#exceptions

Reply:
https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access